exercises/armstrong-numbers/canonical-data.json

```diff
@@ -0,0 +1,86 @@
+{
+  "exercise": "armstrong-numbers",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "0 is an Armstrong number",
+      "property": "isArmstrongNumber",
+      "input": {
+        "number": 0
+      },
+      "expected": true
+    },
+    {
+      "description": "1 is an Armstrong number",
+      "property": "isArmstrongNumber",
+      "input": {
+        "number": 1
+      },
+      "expected": true
+    },
+    {
+      "description": "9 is an Armstrong number",
```

Comment:
Appears as though the first three tests could be consolidated into a single test since all single digit numbers are Armstrong numbers.

```json
"description": "Single digit numbers are Armstrong numbers",
```
My wording may need some work.  I don't want to imply that all single digit number need to be checked.